docs/Config.md

```diff
@@ -12,9 +12,19 @@ your slack app (api.slack.com/apps).
 
 ## SLACK\_API\_TOKEN
 
-The slack api token of your slack bot. Can be found under OAuth & Permissions
+The Slack API token of your Slack bot. Can be found under OAuth & Permissions
 tab of your slack app (under the name "Bot user OAuth access token").
 
+The following permission scopes are required:
+
+- `channels:read`
+- `channels:manage`
+- `chats:write`
+- `incoming-webhooks`
+- `users.profile:read`
+- `users:read`
+- `commands`
```

Comment:
atm these are just best guesses, not sure what is actually needed

Reply:
Really good guesses. Also add:

- `groups:read` (for private channels, optionally, for the random channel promoter)
- `im:write` (for chatting with the people who just joined on how to get set up)

Reply:
98da49f